dist/starter/resources/archetypes/kubernetes-archetype/src/main/resources/archetype-resources/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.1/apache-maven-3.6.1-bin.zip
```

Comment:
I'd better use the latest available v3.6.3 if possible.

Reply:
Yes. I'll increase the version to v3.6.3